docs/developers/setup-documentation.rst

```diff
@@ -18,13 +18,17 @@ Setting up
 This presumes that you're familiar with the command line, and are using OSX or another UNIX-like system.
```

Comment:
That's fine, go ahead and just stub that page out with the links you've collected there and we can flesh it out as we go